docs/content/integrations/dbt/reference.mdx

```diff
@@ -186,6 +233,7 @@ daily_dbt_assets_schedule = build_schedule_from_dbt_selection(
     job_name="daily_dbt_models",
     cron_schedule="@daily",
     dbt_select="tag:daily",
+    config=RunConfig(ops={"my_dbt_assets": MyDbtConfig(full_refresh=True)}),
```

Comment:
I think this should be commented out, since this isn't necessary.

Something like,

```suggestion
    # If your definition of `@dbt_assets` has Dagster Configuration, you can specify it here.
    # config=RunConfig(ops={"my_dbt_assets": MyDbtConfig(full_refresh=True)}),
```